docs/_posts/2017-09-26-react-v16.0.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "React v16.0"
+author: [acdlite]
+---
+
+We're excited to announce the release of React v16.0! Among the changes are some long-standing feature requests, including [**fragments**](#new-render-return-types-fragments-and-strings), [**error boundaries**](#better-error-handling), [**portals**](#portals), support for [**custom DOM attributes**](#support-for-custom-dom-attributes), improved [**server-side rendering**](#better-server-side-rendering), and [**reduced file size**](#reduced-file-size).
+
+### New `render` return types: fragments and strings
+
+You can now return an array of elements from a component's `render` method:
+
+```js
+render() {
+  // No need to wrap list items in an extra element!
+  return [
+    // Don't forget the keys :)
+    <li key="A"/>First item</li>,
+    <li key="B"/>Second item</li>,
+    <li key="C"/>Third item</li>,
+  ];
+}
+```
+
+Support for returning strings from the render method has also been added.
+
```

Comment:
Should we mention we are considering adding a syntax that wouldn't require keys in the future? I suspect it will be the most common question.

Reply:
Does this also mean that it is _always_ safe to return `this.props.children` in render, without wrapping it with `React.Children.only`?

Reply:
Yea.